dynawo/sources/Models/Modelica/Dynawo/Electrical/StaticVarCompensators/SVarCPV.mo

```diff
@@ -18,6 +18,10 @@ model SVarCPV "PV static var compensator model"
   import Dynawo.Connectors;
   import Dynawo.Electrical.StaticVarCompensators.BaseControls.Mode;
   import Dynawo.Electrical.SystemBase;
+  import Dynawo.Electrical.Controls.Basics.SwitchOff;
+
+  extends AdditionalIcons.Shunt;
```

Comment:
If you are motivated you can create an icon for svarc, same as the shunt icon with an arrow to show it is variable :D

Reply:
Ok I do it !

Reply:
Thanks! :+1: